index.js

```diff
@@ -422,10 +419,6 @@ Router.prototype.findRoute = function findNode (method, path, constraints = {})
     pattern = pattern.toLowerCase()
   }
 
-  if (pattern === '*') {
```

Comment:
Uncovered branch.

Can't find a good reason why `pattern` could be `*` using apis.